### ISSUE TITLE
Support Arel::Nodes::NullsFirst in GraphQL::Pro::Order::RelationConnection::Order#normalize

### ISSUE THREAD

**Issue by Hugo-Hache** [rmosolgo/graphql-ruby#4910]
**Is your feature request related to a problem? Please describe.**

StableRelationConnection is not able to reverse order on an order composed of NullsFirst Arel node.

**Describe the solution you'd like**

Support NullsFirst 

**Additional context**

Looks to me it would require parsing recursively Node in normalize, ie replace:

``` ruby
            when Arel::Nodes::Ascending, Arel::Nodes::Descending
              dir = v.is_a?(Arel::Nodes::Ascending) ? :asc : :desc
              order_by = v.expr
              name, table = case order_by
              when Arel::Nodes::SqlLiteral
                [order_by, false]
              when Arel::Attributes::Attribute
                [order_by.name.to_s, order_by.relation.name]
              else
                [order_by.to_sql, false]
              end
              normalized_orders << NormalizedOrder.new(name, table, dir, nil, nil, nil)
```

by:

``` ruby
            when Arel::Nodes::Ordering
              normalized_orders << normalize_arel_node_ordering(v)
              
...

  def normalize_arel_node_ordering(node, nulls_last: nil)
    case node
    when Arel::Nodes::NullsFirst, Arel::Nodes::NullsLast
      normalize_arel_node_ordering(node.expr, nulls_last: ### not sure what to do here ###)
    when Arel::Nodes::Ascending, Arel::Nodes::Descending
      dir = v.is_a?(Arel::Nodes::Ascending) ? :asc : :desc
      order_by = v.expr
      name, table = case order_by
      when Arel::Nodes::SqlLiteral
        [order_by, false]
      when Arel::Attributes::Attribute
         [order_by.name.to_s, order_by.relation.name]
      else
        [order_by.to_sql, false]
      end
            
      NormalizedOrder.new(name, table, dir, nulls_last, nil, nil)
    end
  end
```

**Comment by rmosolgo** [comment]
Hey, thanks for the suggestion! I just released graphql-pro v1.27.0 with support for `NullsFirst` and `NullsLast`. Please give that version a try and let me know if you have any trouble with it.

**Comment by Hugo-Hache** [comment]
Works like a charm, thank you!
